<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Alexanders-MacBook-Pro.local" failures="1" tests="3" name="login Test" time="23.757" errors="0" timestamp="2020-10-22T15:57:44 CST">
  <testcase classname="Tests.LoginTest" name="doLogin" time="7.778"/>
  <testcase classname="Tests.LoginTest" name="doLogin" time="2.451"/>
  <testcase classname="Tests.LoginTest" name="doNotAllowUsersToLogin" time="8.153">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at Tests.LoginTest.doNotAllowUsersToLogin(LoginTest.java:65)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- doNotAllowUsersToLogin -->
</testsuite> <!-- login Test -->
